join.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
+    <title>Real Dev Squad | Join</title>
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="stylesheet" href="./css/animate.css" />
+    <link rel="stylesheet" href="./css/navbar.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- Primary Meta Tags -->
+    <meta name="title" content="Real Dev Squad" />
+    <meta
+      name="description"
+      content="Community of developers who collaborate to build Real software."
+    />
+
+    <!-- Open Graph / Facebook Meta Tags -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://www.realdevsquad.com/" />
+    <meta property="og:title" content="Real Dev Squad" />
+    <meta
+      property="og:description"
+      content="Community of developers who collaborate to build Real software."
+    />
+    <meta
+      property="og:image"
+      content="https://raw.githubusercontent.com/Real-Dev-Squad/website-www/develop/img/Real-Dev-Squad-preview.png"
+    />
+
+    <!-- Twitter Meta Tag-->
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:site" content="@RealDevSquad" />
+    <meta property="twitter:title" content="Real Dev Squad" />
+    <meta
+      property="twitter:description"
+      content="Community of developers who collaborate to build Real software."
+    />
```

Comment:
Can we please change it according to the changes to be done in Primary Meta Tags ?